teams/templates/distribute_teams.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load static %}
+{% load teams_tags %}
+{% block content %}
+
+<div class="h-100">
+    <article class="card shadow">
+        <div class="card-body">
+
+        {% if teams %}
+        {% for team_name, team_members in teams.items %}
+        <section class="card shadow">
+            <div class="card-header">{{team_name}}  <span style="float:right">Team Score: {{ team_members|calculate_team_level }}</span></div>
+            <div class="card-body">
+                <ol id="{{team_name}}" ondrop="drop(event)" ondragover="allowDrop(event)">
+                    {% for member in team_members %}
+                    <li id="{{team_name}}-{{ forloop.counter0 }}" draggable="true" ondragstart="drag(event)">{{member.name}} (level: {{member.level}})</li>
+                    {% endfor %}
+                </ol>
+            </div>
+        </section>
+        {% endfor%}
+        {% endif %}
+
+        {% if leftover_participants %}
+        <section class="card shadow">
+            <div class="card-header">Leftover Participants</div>
+            <div class="card-body">
+                <ol ondrop="drop(event)" ondragover="allowDrop(event)">
+                    {% for participant in leftover_participants %}
+                    <li id="leftover_participants-{{ forloop.counter0 }}" draggable="true" ondragstart="drag(event)">{{participant.name}} (level: {{participant.level}})</li>
+                    {% endfor %}
+                </ol>
+            </div>
+        </section>
+        {% endif%}
+        
+        </div>
+    </article>
+</div>
+{% endblock %}
+
+{% block js %}
+{{ teams|json_script:"teams" }}
+{{ leftover_participants|json_script:"leftover_participants" }}
+<script src="{% static 'js/teams.js' %}"></script>
+{% endblock %}
+
+
```

Comment:
Only 1 blank line at end of file needed.

Reply:
Removed the extra blank lines